Tuto-GUDHI-cover-complex.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>This is not portable, we should find a better way to do so</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
I don't know if it is the best way, but I copy pasted the pieces of code in utils (so now there is a little redundancy with Gudhi)